cron/hourly/00-charge-recurring-contributions.js

```diff
@@ -41,13 +42,22 @@ const csvFields = [
 
 const startTime = new Date();
 
+const JOB_RUNNING_KEY = 'cron-hourly-charger-recurring-contributions-running';
+
 if (parseToBoolean(process.env.SKIP_CHARGE_RECURRING_CONTRIBUTIONS) && !process.env.OFFCYCLE) {
   console.log('Skipping because SKIP_CHARGE_RECURRING_CONTRIBUTIONS is set.');
   process.exit();
 }
 
 /** Run the script with parameters read from the command line */
 async function run(options) {
+  if (await sessionCache.get(JOB_RUNNING_KEY)) {
+    console.log('Skipping because job is already running.');
+    process.exit();
+  }
+
+  await sessionCache.set(JOB_RUNNING_KEY, 1, 60 * 60 * 24); // release the lock after one day max
+
```

Comment:
Would be safer to use a [GETSET](https://redis.io/docs/latest/commands/getset/) to make 100% sure we avoid concurrency issues